scripts/compare_benchmarks.py

```diff
@@ -347,9 +347,9 @@ def double_vertical_separators(lines, vertical_separators_to_duplicate):
         + "```diff\n"
     )
     for line in table_string.splitlines():
-        if (green_control_sequence + "+" in line) or ("| Sum " in line and green_control_sequence in line):
```

Comment:
@Bouncner I don't know why we thought that we needed the color control sequence **and** the `+`/`-`, any idea? I believe it should work without it. I updated the benchmark output in the PR's comments and it seems fine to me.

Reply:
Can't think of a reason right now.